integration/hive/src/main/scala/org/apache/carbondata/hiveexample/HiveExample.scala

```diff
@@ -193,5 +196,4 @@ object HiveExample {
     }
     hiveEmbeddedServer2.stop()
```

Comment:
please fix the issue: the hiveExample don't stop after running the code. HiveExample should stop.

Reply:
I have already solved hiveExample stop problem with system.exit()